### ISSUE TITLE
Please add support for variable fonts

### ISSUE THREAD

**Issue by ctrlcctrlv** [sile-typesetter/sile#931]
Default EB Garamond distribution is now variable...

**Comment by simoncozens** [comment]
This is on my radar. The problem is how to represent variable fonts in PDF. PDF doesn't have any support for VF at all, so the strategy has to be to instantiate them as static instances and embed those. I'm waiting on harfbuzz/harfbuzz#2112 to be able to do that static instantiation.

**Comment by alerque** [comment]
Issue #1182 is a duplicate of this, but because it includes an action plan of sorts I'm going to close this as the dup.

**Comment by ctrlcctrlv** [comment]
That is OK, being the first to complain confers no special privileges, unfortunately. :wink:
